### PR TITLE
Fix for status bar not showing correct amount of items

### PR DESCRIPTION
**Pull request by Costallat**
- Status bar doesn't show full amount of items

Signed-off-by: Renato Foot <costallat@hotmail.com>

### CODE REVIEW

**Comment by gerotib** [comment]
@Costallat I tested and sending items to depot works correctly, but I am unable to **unwrap** items, thats because of `|| !item->isWrapable() `, but when I change it to `|| item->isWrapable() `, then its not possible to **wrap** an item. I wonder if that check is even needed, because it works fine w/o this part.

**Comment by Lycanzito** [comment]
> @Costallat I tested and sending items to depot works correctly, but I am unable to **unwrap** items, thats because of `|| !item->isWrapable() `, but when I change it to `|| item->isWrapable() `, then its not possible to **wrap** an item. I wonder if that check is even needed, because it works fine w/o this part.


When I try to unwrap a item:
image
dont works

when i "use"
it works
image
image


and the item cant be "wrap" again 
image


also itens can be though outside the house
image


But now with this changes, only is possible to "unwrap" only inside the house:
unwrap shows:
image

use shows:
image

**Comment by Costallat** [comment]
@gerotib @Lycanzito 

That's because I am trying to block to unwrap things on walls, to fix the issue #340 , so the change that I made didn't work.

**Comment by Costallat** [comment]
Test again

**Comment by Lycanzito** [comment]
> Test again

I will test it, in a couple of minutes, i give u the feedback

**Comment by Costallat** [comment]
Tks,

So I will keep only the fix of the status bar and the store items sent back to the store when the player leaves the house.

**Comment by Costallat** [comment]
Someone can do a final test with those two fixes? 

Tks,

**Comment by Lycanzito** [comment]
> Someone can do a final test with those two fixes?
> 
> Tks,

Tested.. The itens are going to the inbox, right?
image

File: src/game.cpp [hunk 1]
```diff
@@ -2423,7 +2423,9 @@ void Game::playerWrapableItem(uint32_t playerId, const Position& pos, uint8_t st
 		return;
 	}
 
-	if (!item || item->getClientID() != spriteId || item->hasAttribute(ITEM_ATTRIBUTE_UNIQUEID) || item->isWrapable()) {
+	if (!item || item->getClientID() != spriteId 
+			  || !item->hasAttribute(ITEM_ATTRIBUTE_UNIQUEID) 
+			  || !item->isWrapable()) {
```
Comment (gerotib) [inline src/game.cpp]: I removed this part and then its possible to wrap and unwrap items as it should. Do we really need that check?

Comment (Costallat) [reply src/game.cpp]: Yes, you need to check if the item is wrapable, ou you will be able to use wrap / unwrap on all items, the problem is that method is not working properly, for some reason  it is returning false when the item can be wrap/unwrap

**Comment by Lycanzito** [comment]
Well.. But with these changes, we lost the possibility to wrap the item back, right?

**Comment by Costallat** [comment]
> Well.. But with these changes, we lost the possibility to wrap the item back, right?

Yes, because it only works because the code is wrong, so we need to fix the code and this is the first step

**Comment by Costallat** [comment]
Test now

**Comment by Lycanzito** [comment]
> Test now

What should these changes make?

**Comment by Costallat** [comment]
- Wrap/unwrap using 'wrap'
-  Drops wrap/unwrap using 'use'

**Comment by Lycanzito** [comment]
> * Wrap/unwrap using 'wrap'
> * Drops wrap/unwrap using 'use'

ok, will test it, and give u the feedback soon

**Comment by Lycanzito** [comment]
maybe i've made something wrong.. but..

image

File: docker/server-only/login.py [hunk 2]
```diff
@@ -87,9 +87,9 @@ def do_login(data):
                 {
                     'id': 0,
                     'name': 'Open-TibiaBR',
-                    'externaladdressprotected': '127.0.0.1',
+                    'externaladdressprotected': '149.56.124.82',
```
Comment (gpedro) [inline docker/server-only/login.py]: whoops!

**Comment by majestyotbr** [comment]
Tested, working. Thanks for the fix!

**Pull request merged**